Bookish.Web/wwwroot/lib/bootstrap/dist/css/bootstrap.css

```diff
@@ -75,7 +75,7 @@ hr {
 
 h1, h2, h3, h4, h5, h6 {
   margin-top: 0;
-  margin-bottom: 0.5rem;
+  margin-bottom: 5rem;
```

Comment:
Please add your own css file in wwwroot. You can then import it here with something like:
```
<link rel="stylesheet" href="@Url.Content("~/css/home/copies.css")" asp-append-version="true">
```
You would generally have one css file per page, and you often have a shared file (index.css or site.css) with styling you want to add to all pages, e.g. to format all h1 headers a certain way. To add styling to a certain element you can give that element a class and then add a style for that class to the css file. Let me know if any of that didn't make sense or you have any questions! 🙂